client/sender-app/start/start.js

```diff
@@ -31,6 +36,7 @@
     function startGame() {
       playerMessenger.ready();
       toastr.info(playerUser.getUser() + ' wants to play!');
+      vm.btnStart = true;
```

Comment:
Is vm.btnStart just a flag to indicate the player has clicked the ready button? If so, would be more intuitive to call it something like vm.ready or vm.playerReady. That way it's clear that other elements can reference the flag, not just the button